setup.py

```diff
@@ -1,7 +1,11 @@
 import os
 import subprocess
 import time
-from setuptools import find_packages, setup
+
+import numpy as np
+from Cython.Build import cythonize
```

Comment:
If cython is not installed ahead, an ImportError will be raised.

Reply:
That's why I added it to `setup_requires`

Reply:
This import statement happens before installing packages specified in `setup_requires`.